NEWS

```diff
@@ -30,6 +42,10 @@
           o SipHash
           o ARIA (including TLS support)
       o Significant Side-Channel attack security improvements
+      o Add a new ClientHello callback. Provides a callback interface that gives
+        the application the ability to adjust the nascent SSL object at the
+        earliest stage of ClientHello processing, immediately after extensions
+        have been collected but before they have been processed.
```

Comment:
Keep it a one-liner, please.  If we start copying straight from CHANGES, it becomes ridiculous.  NEWS is meant to be short (one-line or close to) summaries